norsk-henvisningsstandard-for-rettsvitenskapelige-tekster.csl

```diff
@@ -0,0 +1,318 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="nb-NO" demote-non-dropping-particle="display-and-sort" page-range-format="expanded">
+  <info>
+    <title>Norsk henvisningsstandard for rettsvitenskapelige tekster (Norsk - Bokmål)</title>
+    <title-short>Norsk rettsvitenskap</title-short>
+    <id>http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster</id>
+    <link href="http://www.zotero.org/styles/norsk-henvisningsstandard-for-rettsvitenskapelige-tekster" rel="self"/>
+    <link href="https://www.uio.no/for-ansatte/enhetssider/jus/ledelses-og-utvalgsmoter/pmr/moter/2018/pmr5/vedlegg/sak9.pdf" rel="documentation"/>
+    <author>
+      <name>Hans Gunnar Slokvik Lian</name>
+      <email>h.g.s.lian@ub.uio.no</email>
+    </author>
+    <author>
+      <name>Stian Øby Johansen</name>
+      <email>s.o.johansen@jus.uio.no</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="law"/>
+    <summary>Norwegian standard citation style for citing Norwegiand and international legal sources. Based on "Formelle krav til rettsvitenskapelige tekster" (August 2017).</summary>
+    <updated>2021-06-08T10:53:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="nb-NO">
+    <terms>
+      <term name="editor" form="verb-short">red.</term>
+      <term name="translator" form="verb-short">overs.</term>
+      <term name="translator" form="short">overs.</term>
+      <term name="editortranslator" form="verb-short">red. og overs.</term>
+      <term name="editortranslator" form="verb">Redigert og oversatt av</term>
+      <term name="translator" form="short">overs.</term>
+      <term name="and others">m.fl.</term>
+      <term name="open-quote">"</term>
```

Comment:
Really straight quotes? That seems unusual (and ugly...)

Reply:
Definitively not right, no. The new/updated style uses «», which I have now added.